modules/tinymce/src/core/main/ts/api/EditorOptions.ts

```diff
@@ -107,10 +107,7 @@ export interface Options {
    * @param {String} name Name of a registered option.
    * @return {Boolean} True if the option value was successfully set, otherwise false.
    */
-  set: {
-    <K extends keyof EditorOptions>(name: K, value: EditorOptions[K]): boolean;
-    <T>(name: string, value: T): boolean;
-  };
+  set: <K extends string, T>(name: K, value: K extends keyof NormalizedEditorOptions ? NormalizedEditorOptions[K] : T) => boolean;
```

Comment:
This was an intentional type change from `EditorOptions` to `NormalizedEditorOptions` as well. The reason being when setting a value the processor can convert it so the normalized type is what we should be allowing for the setter, not the return type which is what `EditorOptions` is.